api/well-known/schemas/components.yaml

```diff
@@ -8,10 +8,9 @@ components:
         - api-endpoints
       properties:
         public-key:
```

Comment:
```suggestion
        ear-verification-key:
```

Reply:
Done in [86b1a02](https://github.com/veraison/docs/pull/34/commits/86b1a02e0c1b647515f3d49c6b582fe8afa0c771)